<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>207</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>521</width>
    <height>207</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>521</width>
    <height>207</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>ConverterPNG</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res.qrc">
    <normaloff>:/res/prog.ico</normaloff>:/res/prog.ico</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>131</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
    <property name="icon">
     <iconset resource="../res.qrc">
      <normaloff>:/res/start.ico</normaloff>:/res/start.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpOption">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Настройки</string>
    </property>
    <widget class="QCheckBox" name="chkSubdir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Включить в обработку все подкаталоги</string>
     </property>
     <property name="text">
      <string>Подкатологи</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="chkOptimize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Можно отключить оптимизацию (для теста скорости)</string>
     </property>
     <property name="text">
      <string>Оптимизация</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpProgress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>353</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Прогресс</string>
    </property>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>28</y>
       <width>323</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblStatus">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>30</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(85, 0, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpDir">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>479</width>
      <height>58</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Выберите или перетащите каталог</string>
    </property>
    <widget class="QLineEdit" name="inpDir">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>21</y>
       <width>426</width>
       <height>22</height>
      </rect>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDir">
     <property name="geometry">
      <rect>
       <x>441</x>
       <y>20</y>
       <width>23</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Выбор каталога</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/res/folder.ico</normaloff>:/res/folder.ico</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>131</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
    <property name="icon">
     <iconset resource="../res.qrc">
      <normaloff>:/res/stop.ico</normaloff>:/res/stop.ico</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>24</width>
      <height>24</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>btnStop</zorder>
   <zorder>btnStart</zorder>
   <zorder>grpOption</zorder>
   <zorder>grpProgress</zorder>
   <zorder>grpDir</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inpDir</tabstop>
  <tabstop>btnDir</tabstop>
  <tabstop>chkSubdir</tabstop>
  <tabstop>chkOptimize</tabstop>
  <tabstop>btnStart</tabstop>
 </tabstops>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
